admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -63,12 +63,6 @@ $(document).ready(function() {
     $(this).val(parsedValue);
   });
 
-  /** Attach date picker */
```

Comment:
Why removing this datetimepicker? Especially in the product js while the issue is related to the order view page?

Reply:
@jolelievre Because the problem is relative to all datetimepickers, not only the one in the order view page, and this one caused a bug because datetimepickers are already initialized [here](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/admin-dev/themes/new-theme/js/app/utils/datepicker.js#L67)